osf/utils/outcomes.py

```diff
@@ -0,0 +1,26 @@
+from enum import IntEnum
+
+
+class NoPIDError(Exception):
+    pass
+
+
+class ArtifactTypes(IntEnum):
+    '''Labels used to classify artifacts.
+
+    Gaps are to allow space for new value to be added later while
+    controlling for display order.
+
+    PRIMARY value is arbitrarily large as it is an internal-only concept for now
+    '''
+    UNDEFINED = 0
+    DATA = 1
+    CODE = 11
+    MATERIALS = 21
```

Comment:
Looks like you're about to update the API contract for this, but just in case, we'll probably need to do that.

Reply:
Thanks for the reminder!